common-lib/src/cmr/common/api/errors.clj

```diff
@@ -27,8 +27,8 @@
    :body {:errors ["An Internal Error has occurred."]}})
 
 (defn- keyword-path->string-path
-  "Converts a set of keyword field paths into the string equivalent field paths to return to the
-  user."
+  "Converts a set of keyword field paths into the string equivalent field paths
```

Comment:
Nit: Not sure if this is coming from kibit or just an IDE preference, but we agreed on the CMR to a line limit of 100 characters. It looks like this might be 80. We're fine with anyone wrapping at something less than 100, but if it's a kibit setting we should probably set that value to 100.